playground/app.js

```diff
@@ -352,6 +352,14 @@ class App extends Component {
         <div className="col-sm-5">
           {!this.state.form ? null :
             <Form
+              id="test-id"
+              className="testclass big"
+              name="test-name"
+              method="put"
+              target="_blank"
+              autocomplete="off"
+              enctype="multipart/form-data"
+              acceptcharset="ISO-8859-1"
```

Comment:
Sorry, i'm not sure that my English allows you to do this(

Example:

## Form attributes

Form component supports the following html attributes:

```
<Form
  id="edit-form"
  className="form form-wide"
  name: "testName"
  method: "post"
  target: "_blank"
  action: "/users/list"
  autocomplete: "off"
  enctype: "multipart/form-data"
  acceptcharset: "ISO-8859-1"
  schema={} />
```

It's ok?